packages/js/src/browser.ts

```diff
@@ -99,62 +96,8 @@ class Honeybadger extends Client {
   }
 
   public async showUserFeedbackForm(options: Types.UserFeedbackFormOptions = {}) {
-    if (!this.config || !this.config.apiKey) {
-      this.logger.debug('Client not initialized')
-      return
-    }
-
-    if (!this.__lastNoticeId) {
-      this.logger.debug("Can't show user feedback form without a notice already reported")
-      return
-    }
-
-    const global = globalThisOrWindow()
-    if (typeof global.document === 'undefined') {
-      this.logger.debug('global.document is undefined. Cannot attach script')
-      return
-    }
-
-    if (this.isUserFeedbackScriptUrlAlreadyVisible()) {
-      this.logger.debug('User feedback form is already visible')
-      return
-    }
-
-    global['honeybadgerUserFeedbackOptions'] = {
-      ...options,
-      apiKey: this.config.apiKey,
-      endpoint: this.config.userFeedbackEndpoint,
-      noticeId: this.__lastNoticeId
-    }
-
-    this.appendUserFeedbackScriptTag(global, options)
-  }
-
-  private appendUserFeedbackScriptTag(window: typeof globalThis, options: Types.UserFeedbackFormOptions = {}) {
-    const script = window.document.createElement('script')
-    script.setAttribute('src', this.getUserFeedbackSubmitUrl())
-    script.setAttribute('async', 'true')
-    if (options.onLoad) {
-      script.onload = options.onLoad
-    }
-    (global.document.head || global.document.body).appendChild(script)
-  }
-
-  private isUserFeedbackScriptUrlAlreadyVisible() {
-    const global = globalThisOrWindow()
-    const feedbackScriptUrl =this.getUserFeedbackSubmitUrl()
-    for (let i = 0; i < global.document.scripts.length; i++) {
-      const script = global.document.scripts[i]
-      if (script.src === feedbackScriptUrl) {
-        return true
-      }
-    }
-
-    return false
-  }
-
-  private getUserFeedbackSubmitUrl() {
-    return getUserFeedbackScriptUrl(this.getVersion())
+    const form = new BrowserFeedbackForm(this.config, this.logger, this.getVersion());
```

Comment:
I moved feedback form code to its own file to reduce chances of breaking the code removal by rearranging methods.

Reply:
This refactor broke the integration tests. I am working on fixing them.